bin/yaml/tools.yaml

```diff
@@ -27,6 +42,7 @@ tools:
     symlink: ninja
     targets:
       - 1.11.1
+      - 1.12.0
```

Comment:
Why the added `ninja` here? Where is it referenced?

Reply:
Oh, I was matching Windows' deployment; I can add it to the other repo as well.

Reply:
What uses this `ninja` version, out of interest?

Reply:
I just removed it for now.